src/pages/workspaces/workspace/DeleteWorkspaceModal.js

```diff
@@ -7,57 +7,28 @@ import { warningBoxStyle } from 'src/components/data/data-utils'
 import { icon } from 'src/components/icons'
 import { TextInput } from 'src/components/input'
 import Modal from 'src/components/Modal'
-import { Ajax } from 'src/libs/ajax'
 import { bucketBrowserUrl } from 'src/libs/auth'
 import colors from 'src/libs/colors'
-import { reportError, withErrorReportingInModal } from 'src/libs/error'
-import { useCancellation, useOnMount } from 'src/libs/react-utils'
-import { getUser } from 'src/libs/state'
 import * as Utils from 'src/libs/utils'
 import { isAzureWorkspace, isGoogleWorkspace } from 'src/libs/workspace-utils'
-import { isResourceDeletable } from 'src/pages/workspaces/workspace/analysis/runtime-utils'
+import { useDeleteWorkspaceState } from 'src/pages/workspaces/workspace/delete-workspace-modal.state'
```

Comment:
I've never seen us use a file naming convention like this (*.state.ts), except for the test files. Was this recommended to you? I would suggest naming the file `useDeleteWorkspaceState.ts`.

Reply:
I was following the pattern in [this PR](https://github.com/DataBiosphere/terra-ui/pull/3679/files#diff-15e84a0f092a2b7f95b5ba8b83a52bc2658af8bb727a4ecdf08f0bc2effe2d69), I'm fine moving to `useDeleteWorkspaceState.ts`

Reply:
Ah, I hadn't seen that PR. I added a comment to that PR, and we can always name these files back later if necessary.